content/influxdb/cloud/reference/api/influxdb-1x/_index.md

```diff
@@ -14,4 +14,42 @@ related:
   - /influxdb/cloud/query-data/influxql
 ---
 
-{{< duplicate-oss >}}
\ No newline at end of file
+The InfluxDB v2 API includes InfluxDB 1.x compatibility endpoints that work with
```

Comment:
can we just explicitly say `/query` and `/write` in this page? Those are the only "compatibility endpoints" i believe.